server/app/views/components/FieldWithLabel.java

```diff
@@ -280,12 +280,12 @@ public FieldWithLabel setDisabled(boolean disabled) {
   }
 
   /**
-   * Set this field to required. Causes a required question indicator to be displayed when this
-   * field is rendered.
+   * Set this field to required or optional. If required, a required question indicator will be
+   * displayed when this field is rendered.
    */
-  public FieldWithLabel isRequired() {
-    this.required = true;
-    setAriaRequired(true);
+  public FieldWithLabel setRequired(boolean isRequired) {
```

Comment:
Nice rename.

suggestion(none-blocking as not a a regression):
Is there a reason why we need to methods (setRequired and setAriaRequired)?
If the answer is yes, could you expand the comments to make that more clear?

Reply:
Yes, the setRequired one changes the UI and the setAriaRequired doesn't change the UI but just adds the aria label for screen readers. This is useful sometimes when we use multiple fields for the same question and we don't need the asterisk to show up on each and every field.